client/my-sites/plans/current-plan/ecommerce-trial/index.tsx

```diff
@@ -0,0 +1,266 @@
+	const selectedSiteId = useSelector( ( state ) => getSelectedSiteId( state ) ) || -1;
+
+	const { currentPlan, eCommerceTrialDaysLeft, isTrialExpired, eCommerceTrialExpiration } =
+		useSelector( ( state ) => ( {
+			currentPlan: getCurrentPlan( state, selectedSiteId ),
+			isTrialExpired: isECommerceTrialExpired( state, selectedSiteId ),
+			eCommerceTrialDaysLeft: Math.round( getECommerceTrialDaysLeft( state, selectedSiteId ) || 0 ),
+			eCommerceTrialExpiration: getECommerceTrialExpiration( state, selectedSiteId ),
+		} ) );
+
+	const locale = useLocale();
+	const moment = useLocalizedMoment();
+	const translate = useTranslate();
+
+	const [ showAllTrialFeaturesInMobileView, setShowAllTrialFeaturesInMobileView ] =
+		useState( false );
+
+	const viewAllIncludedFeatures = () => {
+		setShowAllTrialFeaturesInMobileView( true );
+	};
+
+	const isMobile = useMediaQuery( '(max-width: 480px)' );
+	const displayAllIncluded = ! isMobile || showAllTrialFeaturesInMobileView;
+
+	const allIncludedFeatures = [
+		{
+			title: translate( 'Priority support' ),
+			text: translate( 'Need help? Reach out to us anytime, anywhere.' ),
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/priority-support.svg',
+			showButton: true,
+			buttonText: translate( 'Ask a question' ),
+		},
+		{
+			title: translate( 'Premium themes' ),
+			text: translate( 'Explore a diverse selection of beautifully designed premium themes.' ),
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/premium-themes.svg',
+			showButton: true,
+			buttonText: translate( 'Browse premium themes' ),
+		},
+		{
+			title: translate( 'Advanced customization' ),
+			text: translate( "Change your store's appearance in a few clicks!" ),
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/advanced-customization.svg',
+			showButton: true,
+			buttonText: translate( 'Design your store' ),
+		},
+		{
+			title: translate( 'Unlimited products' ),
+			text: translate( 'List as many products or services as you’d like and offer subscriptions.' ),
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/unlimited-products.svg',
+			showButton: true,
+			buttonText: translate( 'Add a product' ),
+		},
+		{
+			title: translate( 'Jetpack features' ),
+			text: translate( 'Get auto real-time backups, malware scans, and spam protection.' ),
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/jetpack-features.svg',
+			showButton: true,
+			buttonText: translate( 'Keep your store safe' ),
+		},
+		{
+			title: translate( 'SEO tools' ),
+			text: translate(
+				'Boost traffic with tools that make your content more findable on search engines.'
+			),
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/seo-tools.svg',
+			showButton: true,
+			buttonText: translate( 'Increase visibility' ),
+		},
+		{
+			title: translate( 'Google Analytics' ),
+			text: translate(
+				'Understand visitors and traffic patterns more in depht with Google stats.'
```

Comment:
There is a typo here.
```suggestion
				'Understand visitors and traffic patterns more in depth with Google stats.'
```
Also, I think we could simplify the wording, as it's currently tricky for me to understand. I'd vote for _Understand visitors and traffic patterns in more depth with Google stats._

Maybe consider unwrapping this from translate or picking my language. Either should work. But we should avoid queueing something we don't want to keep for translation.